abovevtt.css

```diff
@@ -273,7 +273,7 @@ div.ct-sidebar__pane-content--is-dark-mode .sidebar-panel-content {
     left: 271px;
     top: 67px;
     right: auto;
-    width: 410px;
+    min-width:  670px;
```

Comment:
I don't like how big this is, but if it's the only way, it's probably fine. Do you think it would be possible to just open the sheets to this size instead?... hmmm. I guess we might need to fake a hover event to get DDB to inject `#db-tooltip-container` 🤔

Reply:
That was the only thing I could think of... I think this is the best approach for now, and if we can figure out how to do it later on, then we can adjust the `min-width`. For now, can you just add a comment next to the `min-width` to explain why we're setting that value? Otherwise, down the road someone will come in and be like "oh sweet, I just need to lower this!" and then accidentally break the hover states.

Reply:
I got it fixed for smaller windows using blank space off screen in the iframe to trick it into thinking it's bigger.